Account/FxAPushMessageHandler.swift

```diff
@@ -62,64 +62,62 @@ extension FxAPushMessageHandler {
 
         // return handle(plaintext: string)
         let deferred = PushMessageResult()
-        RustFirefoxAccounts.startup(prefs: profile.prefs).uponQueue(.main) { _ in
-            RustFirefoxAccounts.reconfig { accountManager in
```

Comment:
The problem in the old code is if this flow happens:
- call `reconfig()`, async it is initializing account manager, and before this completes,
- call `startup()`,  this also initializes the account manager,
- both of these try to call  `RustFirefoxAccounts.shared.accountManager.fill(...)` (i.e. `Deferred.fill(value)`), which is a fatal error, a deferred can only be filled 1x.